routes/outfits.js

```diff
@@ -1,6 +1,6 @@
 const express = require("express");
 const router = express.Router();
-
+const outfitValidation = require("../validation/outfit_validation");
 const gen_outfitData = require("../data/gen_outfit");
 const outfitsData = require("../data/outfits");
 const clothesData = require("../data/clothes");
```

Comment:
Lines 7 - 10 can be removed since they aren't being used in this file

Reply:
Just pushed the file and removed those lines.